src/testMod/resources/data/advancedperipheralstest/computer/tests/peripheraltest.mecrafting.lua

```diff
@@ -0,0 +1,30 @@
+sleep(4)
+bridge = peripheral.wrap("bottom")
+test.assert(bridge, "Peripheral not found")
+
+isOnline = bridge.isConnected()
+test.assert(isOnline, "Bridge is not connected/system is not online")
+
+validEnergy = bridge.getEnergyUsage() > 24
+test.assert(validEnergy, tostring(bridge.getEnergyUsage()) .. "Consumption does not seem right")
```

Comment:
```suggestion
test.assert(validEnergy, tostring(bridge.getEnergyUsage()) .. " Consumption does not seem right")
```

or `string.format` will looks better than string join